docs/react-testing-library/example-intro.md

```diff
@@ -133,8 +147,6 @@ export default function Fetch({ url }) {
 ```
 
 ```jsx
-expect(axiosMock.get).toHaveBeenCalledTimes(1)
-expect(axiosMock.get).toHaveBeenCalledWith(url)
 expect(screen.getByRole('heading')).toHaveTextContent('hello there')
 expect(screen.getByRole('button')).toHaveAttribute('disabled')
```

Comment:
Could you do me a favor and delete all the snapshot stuff at the bottom. Nobody should be doing that for stuff like this. <3

Reply:
Sure. Will delete them tomorrow.